student.rb

```diff
@@ -15,6 +15,6 @@ def play_hooky
 
   def add_classroom=(classroom)
     @classroom = classroom
-    classroom.add_student(self) unless classroom.students.include?(self)
+    classroom.students << self unless classroom.students.include?(self)
```

Comment:
- [ ] (OPTIONAL) The student should be added to the classroom like this 👍 `classroom.students.push(self) unless classroom.students.include?(self)` Kindly correct this 💯